src/index.js

```diff
@@ -296,6 +297,7 @@ class ReactTooltip extends React.Component {
           );
         }
         target.addEventListener('mouseleave', this.hideTooltip, isCaptureMode);
+        target.addEventListener('blur', this.showTooltip, isCaptureMode);
```

Comment:
Shouldn't on blur triggers `hideTooltip` instead? I believe this caused tooltips to be always displayed when we updated to the latest release

Reply:
Good catch, my apologies, let me fix this